pom.xml

```diff
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
-            <version>20230227</version>
```

Comment:
Is this auto-generated or the date of release?

Reply:
I think so yes
`https://mvnrepository.com/artifact/org.json/json/20230227`